src/components/card/card.jsx

```diff
@@ -12,7 +12,7 @@ const Card = (props) => {
         <img src={src} alt={name} width="280" height="175" />
       </div>
       <h3 className="small-movie-card__title">
-        <a className="small-movie-card__link" href="movie-page.html">{name}</a>
+        <Link className="small-movie-card__link" to="/films/:id">{name}</Link>
```

Comment:
```suggestion
        <Link className="small-movie-card__link" to={`/films/{props.id}`}>{name}</Link>
```
самое время воспользоваться id

Reply:
и добавить в описание Card.propTypes сведения о наличии id